parquet-hadoop/src/main/java/org/apache/parquet/hadoop/codec/CleanUtil.java

```diff
@@ -26,45 +26,84 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-/**
- * A Helper class which use reflections to clean up DirectBuffer. It's implemented for
- * better compatibility with both java8 and java9+, because the Cleaner class is moved to
- * another place since java9+.
+/*
```

Comment:
I think, it would be better to keep the javadoc comments with the usual content about the class and add the hint about the inspiration.

Reply:
I've restored the javadocs.

Reply:
Failed to restore the one `*` that makes it javadoc :)

Reply:
Good catch, added the `*` :-)